amt/locale/nl_NL/LC_MESSAGES/messages.po

```diff
@@ -619,11 +620,11 @@ msgstr "Creëer een Algoritme Systeem"
 
 #: amt/site/templates/projects/new.html.j2:23
 msgid "Algorithm System name"
-msgstr "Algoritmekader"
+msgstr "Naam algoritme systeem"
 
 #: amt/site/templates/projects/new.html.j2:26
 msgid "Your algorithm system name here"
```

Comment:
@anneschuth shall we also update the English version:

> Name of the algorithm system

Reply:
Sure, will do